src/Elcodi/Bundle/CartCouponBundle/Resources/config/eventListeners.yml

```diff
@@ -30,6 +30,14 @@ services:
         tags:
             - { name: kernel.event_listener, event: cart_coupon.onapply, method: checkDuplicates, priority: 16 }
 
+    elcodi.event_listener.cart_coupon.stackable_coupon:
```

Comment:
Can you rename this service using elcodi format (3 elements). Something like `elcodi.event_listener.stackable_coupon`